my-webxr-app/src/contexts/PointSelectionContext.tsx

```diff
@@ -10,7 +10,15 @@ import React, { createContext, useContext, useMemo, useState } from "react";
  * setSelectedDataPoint: React State setter function
  */
 interface PointSelectionContext {
-  selectedDataPoint: number | null;
+  selectedDataPoint: {
```

Comment:
a class DataPoint will be nice to have so we don't have to repeatedly define Data Point's required properties every where like this.
Instead, we can have something more type safe and more concise:
`selectedDataPoint: DataPoint | null`

Reply:
Yes, good idea! I will try to fix that up.